### PR TITLE
improved conversion report

### PR DESCRIPTION
**Pull request by its7ARC**
Fixes [#294](https://github.com/cloudspannerecosystem/harbourbridge/issues/294)

- Added percentage of columns mapped cleanly in migration from the source database to Spanner in the conversion report.
- Added percentage of rows mapped in migration from the source database to Spanner in the conversion report.

### CODE REVIEW

File: sources/mysql/report_test.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func TestReport(t *testing.T) {
 		`----------------------------
 Summary of Conversion
 ----------------------------
-Schema conversion: GOOD (most columns mapped cleanly, but some missing primary keys).
+Schema conversion: GOOD (99.99412% of 17006 columns mapped cleanly) + some missing primary keys.
```
Comment (Deep1998) [inline sources/mysql/report_test.go:77]: You might want to limit the precision to 2 decimal places

Comment (its7ARC) [reply sources/mysql/report_test.go:77]: will do.

Comment (its7ARC) [reply sources/mysql/report_test.go]: done

**Review by Deep1998 — APPROVED** [review]
Looks like the initial precision setting was part of the logic, we should revert it to that. Approving anyways.

**Comment by its7ARC** [comment]
> Looks like the initial precision setting was part of the logic, we should revert it to that. Approving anyways.

This will be reverted in the next pull request.

File: .github/workflows/test-coverage.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Google LLC
```
Comment (Deep1998) [inline .github/workflows/test-coverage.yaml:1]: what are these files?

Comment (its7ARC) [reply .github/workflows/test-coverage.yaml:1]: These .github files haven't been added by me. These aren't actual changes, the current state of the Harbourbridge repository already has these files with the same content in them.

**Comment by its7ARC** [comment]
Closing pull request. Will make new pr's, each adding a specific functionality.

**Pull request closed without merging**